manifests/timersengine.pp

```diff
@@ -0,0 +1,81 @@
+# @summary Manages the <code>st2timersengine</code> service.
+#
+# Normally this class is instantiated by +st2::profile::fullinstall+.
+# However, advanced users can instantiate this class directly to configure
+# and manage just the <code>st2timersengine</code> service on a single node.
+# Parameters for this class mirror the parameters in the st2 config.
+#
+# @see https://github.com/StackStorm/st2/blob/master/conf/st2.conf.sample#L337-L343
+#
+# @example Basic usage
+#   include ::st2::timersengine
+#
+# @example Customizing parameters
+#   class { '::st2::timersengine':
+#     enabled  => true,
+#     timezone => 'America/Los_Angeles',
+#   }
+#
+# @param enabled
+#   Specify to enable timer service.
+# @param timezone
+#   Timezone pertaining to the location where st2 is run.
+#
+class st2::timersengine (
+  $enabled  = $::st2::timersengine_enabled,
+  $timezone = $::st2::timersengine_timezone,
+) inherits st2 {
+
+  # st2timersengine was introduced in 2.9.0
+  if ($::st2::version == 'latest' or
+      $::st2::version == 'present' or
+      $::st2::version == 'installed' or
+      versioncmp($::st2::version, '2.9.0') >= 0) {
```

Comment:
Is the `$::st2::version` var here specified by user as an input param or we lookup it during the runtime?

I'm wondering what happens if st2 was already installed in the system and we're doing an upgrade.